spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/autoconfigure/dashscope/DashScopeAutoConfiguration.java

```diff
@@ -78,6 +78,12 @@
 		WebClientAutoConfiguration.class })
 public class DashScopeAutoConfiguration {
 
+	@Bean
+	@ConditionalOnMissingBean
+	public RestClient.Builder restClientBuilder() {
+		return RestClient.builder();
+	}
+
```

Comment:
What's this change for?

Reply:
After removing vaadin, the RestClient.Builder needs to be explicitly injected.
I'll move this change into the example scope.